source/get-response.ts

```diff
@@ -1,21 +1,28 @@
-'use strict';
 const decompressResponse = require('decompress-response');
-const is = require('@sindresorhus/is');
 const mimicResponse = require('mimic-response');
-const progress = require('./progress');
-
-module.exports = (response, options, emitter) => {
+import is from '@sindresorhus/is';
+import {download} from './progress';
+import {IncomingMessage} from 'http';
+import EventEmitter from 'events';
+
+interface Options {
+	decompress: boolean;
+	encoding: any;
```

Comment:
This needs a more explicit type. Are you able to find a Node.js type for it? Look in the Node.js type definitions.

Reply:
yes, I found BufferEncoding,  Should I change this to undefined too? https://github.com/sindresorhus/got/blob/51d72433ccf0de0a9d089c484a3753362af9ac02/source/get-response.ts#L26

Reply:
No, we can't change that (yet), as it's a public API that uses `null`: https://github.com/sindresorhus/got#encoding